Gems/EditorPythonBindings/Code/Include/EditorPythonBindings/PythonUtility.h

```diff
@@ -127,5 +138,57 @@ namespace EditorPythonBindings
 
         //! Calls a BehaviorMethod with a tuple of arguments for member class level functions
         pybind11::object ClassMethod(AZ::BehaviorMethod* behaviorMethod, AZ::BehaviorObject self, pybind11::args args);
-    }
-}
+    } // namespace Call
+
+    namespace Text
+    {
+        class PythonBehaviorDescription
+        {
+        public:
+            //! Get Python type for behavior typeId.
+            AZStd::string_view FetchPythonTypeAndTraits(const AZ::TypeId& typeId, AZ::u32 traits);
+            AZStd::string FetchPythonTypeName(const AZ::BehaviorParameter& param);
+            AZStd::string FetchOutcomeType(const AZ::TypeId& typeId);
+
+            //! Creates a string containing bus events and documentation.
+            AZStd::string BusDefinition(const AZStd::string busName, const AZ::BehaviorEBus* behaviorEBus);
+
+            //! Creates a string with class or global method definition and documentation.
+            //! @param defineTooltip if true, the tooltip will be included in the definition
```

Comment:
I do not see the reason to document one parameter and skip the documentation of the rest. The same goes for other methods in this file.